Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -7363,6 +7363,7 @@ class WebCore::NetworkLoadMetrics {
     bool reusedConnection();
     bool doesFailTAOCheck();
     bool crossOriginRedirect();
+    bool fromPrefetch;
```

Comment:
It's a bit inconsistent to use the accessor methods for the others but access the member directly for this one.  The simplest way to make it consistent would be to add an isFromPrefetch() const accessor, but maybe we could just use [BitField] to encode all the members directly.

Reply:
Moved to [BitField]